src/components/Footer.tsx

```diff
@@ -0,0 +1,82 @@
+import { FaXTwitter, FaLinkedin } from "react-icons/fa6";
+import { FaInstagram } from "react-icons/fa";
+import { BsChat } from "react-icons/bs";
+
+import { footerIcon } from "@assets";
+
+const Footer = () => {
+    return (
+        <div>
+            <hr className="border-t-1 border-gray-200 mt-12 mb-12" />
```

Comment:
LGTM, but maybe copy the color from the other hr? To match the figma

```suggestion
            <hr className="border-t-1 border-[#3D3D3D] mt-12 mb-12" />
```